products/red-hat-openshift.md

```diff
@@ -0,0 +1,145 @@
+    latest: "4.8.57"
+    latestReleaseDate: 2023-01-25
+    extendedSupport: 2023-04-27
+
+-   releaseCycle: "4.7"
+    releaseDate: 2021-02-24
+    support: 2021-10-27
+    eol: 2022-08-24
+    latest: "4.7.60"
+    latestReleaseDate: 2022-11-10
+    extendedSupport: false
+
+-   releaseCycle: "4.6"
+    releaseDate: 2020-10-27
+    support: 2021-03-24
+    eol: 2022-10-27
+    latest: "4.6.62"
+    latestReleaseDate: 2022-11-10
+    extendedSupport: false
+
+-   releaseCycle: "4.5"
+    releaseDate: 2020-07-13
+    support: 2020-11-27
+    eol: 2021-07-27
+    latest: "4.5.41"
+    latestReleaseDate: 2021-06-30
+    extendedSupport: false
+
+-   releaseCycle: "4.4"
+    releaseDate: 2020-05-05
+    support: 2020-08-13
+    eol: 2021-02-24
+    latest: "4.4.33"
+    latestReleaseDate: 2021-02-02
+    extendedSupport: false
+
+-   releaseCycle: "4.3"
+    releaseDate: 2020-01-23
+    support: 2020-06-05
+    eol: 2020-10-27
+    latest: "4.3.40"
+    latestReleaseDate: 2020-10-20
+    extendedSupport: false
+
+-   releaseCycle: "4.2"
+    releaseDate: 2019-10-16
+    support: 2020-02-23
+    eol: 2020-07-13
+    latest: "4.2.36"
+    latestReleaseDate: 2020-07-01
+    extendedSupport: false
+
+-   releaseCycle: "4.1"
+    releaseDate: 2019-06-04
+    support: 2019-11-16
+    eol: 2020-05-05
+    latest: "4.1.41"
+    latestReleaseDate: 2020-04-22
+    extendedSupport: false
+
+---
+
+>[OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift) is a family of
+> containerization software products developed by Red Hat. Its flagship product is the OpenShift
+> Container Platform — a hybrid cloud platform as a service built around Linux containers
+> orchestrated and managed by Kubernetes on a foundation of Red Hat Enterprise Linux.
+
+Red Hat provides three different phases of support: Full Support, Maintenance Support and Extended
+Update Support.
+
+The **Full Support** phase begins at the GA/release of the minor version and ends after a 6 months
+period OR 90 days after the GA of the superseding minor release, whichever is later. During the Full
+Support Phase, qualified Critical and Important Security Advisories ([RHSAs][DEFINITION]) and Urgent
+and Selected High Priority Bug Fix Advisories ([RHBAs][DEFINITION]) will be released as they become
+available; all other available fix and qualified patches may be released via periodic updates.
+
+The **Maintenance Support** phase commences after the Full Support phase for the respective minor
+version and ends at 18 months after GA. During the Maintenance Support phase, qualified Critical and
+Important Security Advisories (RHSAs) and Urgent and Selected High Priority Bug Fix Advisories
+(RHBAs) may be released as they become available. Other Bug Fix and Enhancement Advisories
+([RHEA][DEFINITION]) may be released at Red Hat’s discretion, but should not be expected.
+
+The **Extended Update Support** phase commences after the Full Support phase for even numbered minor
```

Comment:
Should we document that 4.8 is in the Extended life phase, not the Extended update support phase ?

Reply:
If think we shouldn't. That would make the description more complicated for something that only concerns one release.